client/docs/vscode-template/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "files.exclude": {
+    "**/.npm": true,
+    "**/node_modules": true,
+    // "**/dist/": true,
+    // "**/.tsbuildinfo": true,
+    // "**/package-lock.json": true
+  },
```

Comment:
Are you using a JSON extension here? Comments aren't accepted as standard JSON.

Reply:
VSCode accepts comments in its configuration files. Not sure exactly what the rules are for that, but this isn't a syntax error.

Reply:
Oh I see this is VSCode specifically. This is fine then.